### PR TITLE
Adding better OWNERS related PR messages

### PR DESCRIPTION
**Pull request by dperaza4dustbit**
Changing sanity check to add more precise messages around OWNERS file.

### CODE REVIEW

**Comment by dperaza4dustbit** [comment]
Still testing hold merging

**Comment by dperaza4dustbit** [comment]
Test with chart and owners file under partners: https://github.com/dperaza4dustbit/development/pull/1

**Comment by dperaza4dustbit** [comment]
Test with chart and owners under redhat: https://github.com/dperaza4dustbit/development/pull/2

**Comment by dperaza4dustbit** [comment]
Test with chart and owners under community: https://github.com/dperaza4dustbit/development/pull/3

**Comment by dperaza4dustbit** [comment]
Test PR containing owners file only: https://github.com/dperaza4dustbit/development/pull/4

**Comment by mmulholla** [comment]
The new sanity check code puts out two ERROR messages if the non chart file is an OWNERS file.

**Comment by dperaza4dustbit** [comment]
Test with OWNERS file only after a modification to messages: https://github.com/dperaza4dustbit/development/pull/5

**Review by zonggen — COMMENTED** [review]
Looking at the code I think this LGTM.
minor: conside ending with period for [ERROR] messages

File: scripts/src/sanitycheckpr/sanitycheckpr.py [hunk 1]
```diff
@@ -59,6 +60,29 @@ def ensure_only_chart_is_modified(api_url, repository, branch):
...
+            if category == "partners":
+                msg = "[ERROR] OWNERS file should never be set directly by partners. See certification docs"
+                print(msg)
+                print(f"::set-output name=owners-error-message::{msg}")
+            elif match_found: # There is a mix of chart and non-chart files including OWNERS
+                msg = "[ERROR] Send OWNERS file by itself in a PR"
+                print(msg)
+                print(f"::set-output name=owners-error-message::{msg}")
+            elif file_count == 1: # OWNERS is the only file sent on PR
+                msg = "[ERROR] OWNERS file changes require manual review by maintainers"
```
[the remark this answers is not in the document]
Comment (dperaza4dustbit) [reply scripts/src/sanitycheckpr/sanitycheckpr.py]: Was able to only show one info message in the case where only the OWNERs file is sent

[the remark this answers is not in the document]
Comment (dperaza4dustbit) [reply scripts/src/sanitycheckpr/sanitycheckpr.py]: Found a bug here, we need to handle the case, where partners sends pr with only the OWNERS file

[the remark this answers is not in the document]
Comment (dperaza4dustbit) [reply scripts/src/sanitycheckpr/sanitycheckpr.py]: fixed this

**Review by mmulholla — APPROVED** [review]
LGTM

**Review by zonggen — APPROVED** [review]
:+1: as well

**Pull request merged**